tests/templates/test_subroutines/test_qubitization.py

```diff
@@ -138,30 +138,42 @@ def test_legacy_new_opmath():
         (
             qml.ops.LinearCombination(np.array([1.0, 1.0]), [qml.PauliX(0), qml.PauliZ(0)]),
             [
-                qml.AmplitudeEmbedding(np.array([1.0, 1.0]) / np.sqrt(2), wires=[1]),
+                qml.AmplitudeEmbedding(
+                    np.array([1.0, 1.0]) / np.sqrt(2), wires=[1], pad_with=0, normalize=True
+                ),
                 qml.Select(
                     ops=(
                         qml.PauliX(0) @ qml.GlobalPhase(np.array(0.0), wires=0),
                         qml.PauliZ(0) @ qml.GlobalPhase(np.array(0.0), wires=0),
                     ),
                     control=[1],
                 ),
-                qml.adjoint(qml.AmplitudeEmbedding(np.array([1.0, 1.0]) / np.sqrt(2), wires=[1])),
+                qml.adjoint(
+                    qml.AmplitudeEmbedding(
+                        np.array([1.0, 1.0]) / np.sqrt(2), wires=[1], pad_with=0, normalize=True
```

Comment:
Why the new args are needed now?

Reply:
It seems that there was a small bug in `AmplitudeEmbedding` because it was not considering `normalize` and `pad_with` as hyperparameters when they are. When correcting this, this test complained because it had to check that `pad_with` and `normalize` also matched in the decomposition